content/agent/prometheus.md

```diff
@@ -15,11 +15,52 @@ further_reading:
 
 ## Overview
 
-This page first looks at the `PrometheusCheck` interface, and then proposes a simple Prometheus Check that collects timing metrics and status events from [Kube DNS](https://github.com/DataDog/integrations-core/blob/master/kube_dns/datadog_checks/kube_dns/kube_dns.py).
+This page first looks at the generic `Prometheus` check, the fastest & simpliest way to scrap custom metrics off prometheus endpoints and then dives in the `PrometheusCheck` interface for more advanced usage finishing by an example of writting a simple Check that collects timing metrics and status events from [Kube DNS](https://github.com/DataDog/integrations-core/blob/master/kube_dns/datadog_checks/kube_dns/kube_dns.py).
 
-## Prometheus check interface 
+## Generic Prometheus check
 
-`PrometheusCheck` is [a mother class](https://github.com/DataDog/dd-agent/blob/master/checks/prometheus_check.py) providing a structure and some helpers to collect metrics, events, and service checks exposed via Prometheus. Minimal configuration for checks based on this class include:
+Starting with version 6.1 the agent includes a new [Prometheus](https://github.com/DataDog/integrations-core/tree/master/prometheus) check capable of scrapping prometheus endpoints with only a few lines of configuration.
+
+### Configuration
+
+Edit the `prometheus.yaml` file to add your different instances you want to retrieve custom metrics from.
+
+Minimal configuration of an instance includes:
+
+* A `prometheus_url` that points to the metric route (⚠ this has to be unique).
+* A `namespace` that is prepended to all metrics (to avoid metrics name collision).
+* A list of `metrics` that you want to retrieve as custom metrics, for each metric you can either simply add it to the list `- metric_name` or renaming it like `- metric_name: renamed`.
+
+Note: It's possible to use a `*` wildcard such as `- metric*` that would fetch all matching metrics (to use with caution as it can potentially send a lot of custom metrics)
+
+Your metrics are collected in the form of `namespace.metric_name`. By default you get a service check named `namespace.prometheus.health` to indicate the health of the prometheus endpoint.
+
+### Advanced Settings
+
+For a comprehensive list of settings please refer to the [example configuration](https://github.com/DataDog/integrations-core/blob/master/prometheus/conf.yaml.example).
+
+### Auto-discovery
+
+As usual you can configure the prometheus check using [Autodiscovery](https://docs.datadoghq.com/agent/autodiscovery/) if you want to quickly scrap a container/pod exposing prometheus metrics.
```

Comment:
* scrape, not scrap. 😄 

This sentence suggests that the _check_ exposes prometheus metrics, which is clearly not the case. Suggest:
> As usual, if you want to quickly scrape a container/pod that exposes Prometheus metrics, you can configure this check using Autodiscovery.

As an aside, be careful with phrases like "as usual". It's probably OK to use it here, but in general, these sorts of leading phrases should be used judiciously.